api/app/DoctrineMigrations/Version237.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version237 extends AbstractMigration
+{
+    public function getDescription() : string
+    {
+        return 'Create court_order rows for all existing lay clients';
+    }
+
+    public function up(Schema $schema) : void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'postgresql', 'Migration can only be executed safely on \'postgresql\'.');
+
+        $this->addSql("
+insert into court_order (client_id, type, supervision_level, order_date, case_number)
+select 
+  distinct(c.id),
+  case
+    -- if client does not have a report (only has an NDR) then type at this point is NULL
+    when r.id is null then null
+    -- otherwise assign HW for any '4' based reports, and PFA for all others
+    when r.type = '102_4' 
```

Comment:
dunno if it's more clear or not but you can do r.type in ('102_4', 103_4', etc)

Reply:
Good point, cheers